src/CustomErrors.bas

```diff
@@ -5,7 +5,7 @@ Attribute VB_Description = "Global, general-purpose procedures involving run-tim
 Option Explicit
 Option Private Module
 
-Public Const CustomError As Long = vbObjectError Or 32
+Public Const CustomError As Long = vbObjectError Or 32 'QUESTION: VF: why this value?

```

Comment:
Any value works, the idea is to make custom errors enumerate from a (very) negative value to easily tell them apart from "built-in" error codes; `vbObjectError` alone does that, so the offset serves to tell *these* errors apart from custom errors in other code... which admittedly is questionably useful in VBA.